src/main/resources/fix_mod_consortia_table.sql

```diff
@@ -0,0 +1,20 @@
+DO $$
+DECLARE
+    target_tenant TEXT:= 'consortium';
```

Comment:
need to swap, in their example targetTenant it is member tenant

Reply:
better to rename param in our script as well to not have confusion to 'memberTenant'

Reply:
changed and used your suggestion